ggml.h

```diff
@@ -657,8 +668,9 @@ extern "C" {
 
     GGML_API void    ggml_print_backtrace(void);
 
-    GGML_API void    ggml_numa_init(void); // call once for better performance on NUMA systems
-    GGML_API bool    ggml_is_numa(void); // true if init detected that system has >1 NUMA node
+    GGML_API void       ggml_numa_init(uint32_t numa); // call once for better performance on NUMA systems
+    GGML_API bool       ggml_is_numa(void); // true if init detected that system has >1 NUMA node
+    GGML_API cpu_set_t  ggml_get_numa_affinity(void); // get cpuset from numactl
```

Comment:
No need to expose this in the public API. Also remove the `<sched.h>` header from `ggml.h`